src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Avoid using the `*` selector as it impacts performance. It's better to set styles only for the elements that require them. Zeroing out your margins, paddings or other styles with `*` is still inefficient for the browser.